digiwf-connector/digiwf-connector-starter/src/test/java/de/muenchen/oss/digiwf/connector/core/application/usecase/CreateIncidentUseCaseTest.java

```diff
@@ -0,0 +1,34 @@
+package de.muenchen.oss.digiwf.connector.core.application.usecase;
+
+import de.muenchen.oss.digiwf.connector.core.application.port.out.CreateIncidentOutPort;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+
+class CreateIncidentUseCaseTest {
+
+    private CreateIncidentUseCase useCase;
+    private CreateIncidentOutPort outPort;
+
+    @BeforeEach
+    void setUp() {
+        outPort = mock(CreateIncidentOutPort.class);
+        useCase = new CreateIncidentUseCase(outPort);
+    }
+
+    @Test
+    void createIncident_shouldCallOutPortWithSameArguments() {
+        // Arrange
+        String processInstanceId = "123";
+        String messageName = "testMessage";
+        String messageContent = "Test Content";
+
+        // Act
+        useCase.createIncident(processInstanceId, messageName, messageContent);
+
+        // Assert
+        verify(outPort).createIncident(processInstanceId, messageName, messageContent);
```

Comment:
Wenn dann sollten die Kommentare überall konsistent sein. Bei der anderen Tests war es given when then

```suggestion
        // Given
        String processInstanceId = "123";
        String messageName = "testMessage";
        String messageContent = "Test Content";

        // When
        useCase.createIncident(processInstanceId, messageName, messageContent);

        // Then
        verify(outPort).createIncident(processInstanceId, messageName, messageContent);
```